ansible/roles/stack-sunbird/templates/sunbird_notification-service.env

```diff
@@ -1,3 +1,14 @@
+{% if (cassandra_cluster_size | int) > 1 %}
+sunbird_cassandra_host={{groups['cassandra']|join(',')}}
+sunbird_cassandra_port=9042,9042,9042
+sunbird_cassandra_consistency_level=quorum
+{% else %}
+sunbird_cassandra_host={{sunbird_cassandra_host}}
```

Comment:
We can not use the variables which are present in if condition. Ansible will fail the task stating that **AnsibleUndefinedVariable**. Can you update the variables with exact value which are in else block.  @sknirmalkar89